API.md

```diff
@@ -0,0 +1,259 @@
+     -X PATCH \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": { "id": "2140", "type": "networks", "attributes": { "name": "A new network" } } }'
+```
+
+#### Network Users
+
+##### List network users
+
+```
+curl http://greencommons.herokuapp.com/api/v1/networks/2140/relationships/users
+```
+
+Note that it is also possible to just include the members when retrieving a network:
+
+```
+curl 'http://greencommons.herokuapp.com/api/v1/networks/2140?include=users'
+```
+
+##### Add users to a network
+
+```
+curl http://greencommons.herokuapp.com/api/v1/networks/2140/relationships/users \
+     -X POST \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": [{ "id": "1", "type": "users" }] }'
+```
+
+##### Remove users from a network
+
+```
+curl http://greencommons.herokuapp.com/api/v1/networks/2140/relationships/users \
+     -X DELETE \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": [{ "id": "1", "type": "users" }] }'
+```
+
+#### Lists
+
+##### Retrieve a list - `/api/v1/lists/:id`
+
+__Example__
+
+```
+curl http://greencommons.herokuapp.com/api/v1/lists/93
+```
+
+##### Create a list
+
+```
+curl http://greencommons.herokuapp.com/api/v1/lists \
+     -X POST \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": { "type": "lists", "attributes": { "name": "A new list" } } }'
+```
+
+##### Add to a list
+
+```
+curl http://greencommons.herokuapp.com/api/v1/lists/93/relationships/items \
+     -X POST \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": [{ "id": "2140", "type": "networks" }, { "id": "43130", "type": "resources" }] }'
+```
+
+##### Remove from a list
+
+```
+curl http://greencommons.herokuapp.com/api/v1/lists/93/relationships/items \
+     -X DELETE \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": [{ "id": "2140", "type": "networks" }, { "id": "43130", "type": "resources" }] }'
+```
+
+#### Users
+
+##### Create users
+
+```
+curl http://greencommons.herokuapp.com/api/v1/users \
+     -X POST \
+     -H 'Authorization: GC SOcdhU1l5Jiebg6EDtyDFQ:1168c1433a2e7aa5945401e7245186e3' \
+     -H 'Content-Type: application/vnd.api+json' \
+     -d '{ "data": { "type": "users", "attributes": { "email": "john@example.com", "password": "password", "password_confirmation": "password"} } }'
+```
+
+##### Update user
+
+```
+curl http://greencommons.herokuapp.com/api/v1/users/10 \
```

Comment:
@ptrikutam Since I'm using the keys from that account to connect to the API, it's the only one I can update (unless I create a new account and update all the access/secret keys).

Waiting on the conclusion about using actual IDs or not before doing it ;)